proto/ibc/applications/interchain_accounts/controller/v1/tx.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package ibc.applications.interchain_accounts.controller.v1;
+
+option go_package = "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types";
+
+import "gogoproto/gogo.proto";
+
+// Msg defines the 27-interchain-accounts/controller Msg service.
+service Msg {
+  // RegisterAccount defines a rpc handler for MsgRegisterAccount.
+  rpc RegisterAccount(MsgRegisterAccount) returns (MsgRegisterAccountResponse);
+}
+
+// MsgRegisterAccount defines the payload for Msg/RegisterAccount
+message MsgRegisterAccount {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string connection_id = 1 [(gogoproto.moretags) = "yaml:\"connection_id\""];
+  string owner         = 2;
+  string version       = 3;
+}
+
+// MsgRegisterAccountResponse defines the response for Msg/RegisterAccount
+message MsgRegisterAccountResponse {}
```

Comment:
Is it useful to return the portID? I guess not since the generation of the portID is stateless?

Reply:
I think If we're going to add the channelID we may need to break the `RegisterInterchainAccount` API to return it? 
That is, depending on how we approach this...

I'm just thinking it would be need to be retrieved from the `MsgChannelOpenInitResponse` from `sdk.Result.MsgResponses[0]`: https://github.com/cosmos/ibc-go/blob/main/modules/apps/27-interchain-accounts/controller/keeper/account.go#L44

What do you think?

Reply:
Why do we need to break the `RegisterInterchainAccount` API?

I was thinking we would make a private function `registerInterchainAccount` (with a slightly diff name) which both `RegisterInterchainAccount` and the rpc `RegisterAccount` call into, that way we can add more info to the API

Reply:
Great point! That sounds good to me! :)